docs/overview/office-add-ins.md

```diff
@@ -1,152 +1,130 @@
 
->**Note:** Content add-ins do not currently support add-in commands.
+**Components of a Hello World Office Add-in**
 
-**Content add-in**
+![Components of a Hello World add-in](../../images/DK2_AgaveOverview07.png)
 
-![In content add-in](../../images/DK2_AgaveOverview05.png)
+## Extending and interacting with Office clients 
 
-To try out a content add-in in Excel 2013 or Excel Online, install the [Bing Maps](https://store.office.com/bing-maps-WA102957661.aspx?assetid=WA102957661) add-in.
+Office Add-ins can do the following within an Office host application: 
 
-### Outlook add-ins that extend functionality
+-  Extend functionality (any Office application) 
 
-Outlook add-ins can extend the Office ribbon and also display contextually next to an Outlook item when you're viewing or composing it. They can work with an email message, meeting request, meeting response, meeting cancellation, or appointment in a read scenario - the user viewing a received item - or in a compose scenario - the user replying or creating a new item. 
+-  Create new objects (Excel or PowerPoint) 
+ 
+### Extend Office functionality 
 
-Outlook add-ins can access contextual information from the item, such as address or tracking ID, and then use that data to access additional information on the server and from web services to create compelling user experiences. In most cases, an Outlook add-in runs without modification on the various supporting host applications, including Outlook, Outlook for Mac, Outlook Web App, and OWA for Devices, to provide a seamless experience on the desktop, web, and tablet and mobile devices.
+You can add new functionality to Office applications via the following:  
 
-To learn more, see [Outlook add-ins](../outlook/outlook-add-ins.md).
+-  Custom ribbon buttons and menu commands (collectively called “add-in commands”) 
 
- >**Note**  Outlook add-ins require a minimum version of Exchange 2013 or Exchange Online to host the user's mailbox. POP and IMAP email accounts aren't supported.
+-  Insertable task panes 
 
-**Outlook add-in with command buttons on the ribbon**
+Custom UI and task panes are specified in the add-in manifest.  
 
-![Add-in Command](../../images/41e46a9c-19ec-4ccc-98e6-a227283623d1.png)
+#### Custom buttons and menu commands  
 
-**Contextual Outlook add-in**
+You can add custom ribbon buttons and menu items to the ribbon in Office for Windows Desktop and Office Online. This makes it easy for users to access your add-in directly from their Office application. Command buttons can launch different actions such as showing a task pane with custom HTML or executing a JavaScript function.  
 
-![Contextual add-in](../../images/DK2_AgaveOverview06.png)
+We recommend that you [watch this Channel9 video](https://channel9.msdn.com/events/Build/2016/P551) for a deeper overview of this feature. 
```

Comment:
This link won't make as much sense out of context, but I think that we can delete it from the Overview topic; we should be linking to the video in the add-in commands topic.